docs/content/workflows/rna_seq.md

```diff
@@ -128,22 +128,78 @@ To do so, you can add the color by name (google "matplotlib colors" for the opti
 The following section will guide you through the process of creating a DESeq2 contrast using only the samples.tsv and the config.yaml.
 
 A contrast contains a **condition**, which is the variable you wish to analyse, and which contains 2 or more states. Optionally, a contrast can contain a **batch** effect, which you want to correct for.
+To determine differentially expressed genes, DESeq2 requires at least two samples per states. A design contrast therefore requires at least 2x2 samples.
 
 #### Conditions
-In samples.tsv, add a column for every property of your samples.
-Examples:
-* a column named 'conditions' with values ‘control’ and ‘treated’.
-* a column named 'conditions' with values ‘control’, ‘treatmentA’ and ‘treatmentB’.
-* a column named 'stages' with values 1, 2 and 3.
+In the samples.tsv, add a column for every property you wish to test in your samples. 
+Next, add labels to the samples involved in the test. You can leave labels empty, or add labels and not use them. 
+For example:
 
-Next, in the `config.yaml` add one or more contrasts.
+1. a column named 'conditions' with values ‘wildtype’ and ‘knockout’.
+2. a column named 'treatments' with values ‘control’, ‘treatmentA’ and ‘treatmentB’.
+3. a column named 'stages' with values 1, 2 and 3.
 
-In order to compare two groups from a list (e.a. stage 1 vs stage 3), the contrast condition is the column name, followed by the groups (`stages_3_1`). Note that the *last* group (stage 1 in this example) is the reference group.
+<table>
+<tr><th>Example 1</th><th>Example 2</th><th>Example 3</th></tr>
+<tr><td>
 
-To compare all groups against a reference (e.a. control vs both treatments A and B), the contrast condition is the column name, followed by the reference group (`condition_control`). Alternatively, group ‘all’ may also be added (`condition_all_control`).
+|sample|assembly|conditions|
+|---|---|---|
+|sample_1|hg38|wildtype|
+|sample_2|hg38|knockout|
+|sample_3|hg38|wildtype|
+|sample_4|hg38|knockout|
+|sample_5|hg38||
+|sample_6|hg38|unused|
+
+</td><td>
+
+|sample|assembly|stages|
+|---|---|---|
+|sample_1|hg38|1|
+|sample_2|hg38|1|
+|sample_3|hg38|2|
+|sample_4|hg38|2|
+|sample_5|hg38|3|
+|sample_6|hg38|3|
+
+</td><td>
+
+|sample|assembly|treatments|
+|---|---|---|
+|sample_1|hg38|control|
+|sample_2|hg38|control|
+|sample_3|hg38|treatmentA|
+|sample_4|hg38|treatmentA|
+|sample_5|hg38|treatmentB|
+|sample_6|hg38|treatmentB|
+
+</td></tr>
+</table>
```

Comment:
Not sure if this works. Did you test if this build properly?

Reply:
[It works in git md](https://github.com/vanheeringen-lab/seq2science/blob/a8964977cb5d9d18b6f3a22cb4eebe1fff8567c3/docs/content/workflows/rna_seq.md#conditions), any chance it then doesn't work in the github action?

Reply:
Yes, because of markdown also having many flavours, and we convert it to html with some obscure sphinx integration :shrug:

Reply:
That did indeed not work in sphinx. Good catch!